pkg/spdk/server.go

```diff
@@ -251,19 +251,19 @@ func (s *Server) verify() (err error) {
 		func() {
 			r.RLock()
 			defer r.RUnlock()
-		if r.ChainLength < 2 {
-			delete(replicaMap, replicaName)
+			if r.ChainLength < 2 {
```

Comment:
Remove the below changes from this commit

Reply:
Use your change instead.